grafana-plugin/src/containers/AddResponders/parts/AddRespondersPopup/AddRespondersPopup.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getAddRespondersPopupStyles = (theme: GrafanaTheme2) => {
+  return {
+    addRespondersDropdown: css`
+             max-height: 500px;
+        overflow: hidden;
+        border: 1px solid ${theme.colors.border.medium};
+        position: absolute;
+        right: 16px;
+        top: 55px;
+        background: ${theme.colors.background.primary}
+        width: 340px;
+        z-index: 10;
+        `,
```

Comment:
seems like some wrong formatting

Reply:
Was missing a semi-colon on the `background` property, that fixes it, otherwise prettier wasn't doing any formatting